integrations/integration-vip-config.php

```diff
@@ -82,20 +82,31 @@ protected function get_vip_config_from_file( string $slug ) {
 		$config_file_name      = $slug . '-config.php';
 		$config_file_path      = $config_file_directory . '/' . $config_file_name;
 
+		/**
+		 * Temporarily disable warnings by clearing the E_WARNING bit.
+		 *
+		 * Reading config file via `file_get_contents()` even with `@` operator generate warnings on PHP 8+.
+		 */
+		$current_error_reporting = error_reporting(); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_error_reporting
+		error_reporting( $current_error_reporting & ~E_WARNING ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_error_reporting
+
 		/**
 		 * Clear cache to always read data from latest config file.
 		 *
 		 * Kubernetes ConfigMap updates the file via symlink instead of actually replacing the file and
 		 * PHP cache can hold a reference to the old symlink that can cause fatal if we use require
 		 * on it.
 		 */
-		if ( false === @file_get_contents( $config_file_path ) ) { // phpcs:ignore WordPress.PHP.NoSilencedErrors.Discouraged
+		if ( false === file_get_contents( $config_file_path ) ) {
```

Comment:
Another solution is to avoid this `if` statement and always read latest config from file.

Personally I am more inclined toward this solution because its simpler and always reading simple configs from file doesn't seems like an expensive operation.

Reply:
> always read latest config from file

Do you mean just running `clearstatcache` and then `require`? If so, we should do a quick benchmark to see the impact before doing this.

Reply:
Did some quick benchmarking on local machine and here are the stats:

1. `clearstatcache()` with disable warning and if statement is on average taking 2.5 * E-4 seconds
1. `clearstatcache()` without if statement is on average taking 4.0 * E-5 seconds 


```php
$start_time = microtime(true);

// Code here

$end_time = microtime(true);
$execution_time = $end_time - $start_time;

echo "Execution Time: {$execution_time} seconds";
```

Based on these benchmarks removing the if statement and making the code simpler.